### PR TITLE
Fix docs code for ScaleIntensityRangePercentiles

### PR DESCRIPTION
**Pull request by matt3o**
When running the docs code for `ScaleIntensityRangePercentiles`, it failed with the error (MONAI 1.2.dev2316):
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/mhadlich/anaconda3/envs/monai/lib/python3.8/site-packages/monai/transforms/intensity/array.py", line 998, in __call__
    img_t = self._normalize(img=img_t)
  File "/home/mhadlich/anaconda3/envs/monai/lib/python3.8/site-packages/monai/transforms/intensity/array.py", line 971, in _normalize
    a_min: float = percentile(img, self.lower)  # type: ignore
  File "/home/mhadlich/anaconda3/envs/monai/lib/python3.8/site-packages/monai/transforms/utils_pytorch_numpy_unification.py", line 121, in percentile
    result = torch.quantile(x, q, dim=dim, keepdim=keepdim)
RuntimeError: quantile() input tensor must be either float or double dtype
```

This commit updates the docs for the function to use `torch.Tensor` instead.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [x] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: monai/transforms/intensity/array.py [hunk 1]
```diff
@@ -1132,7 +1132,7 @@ class ScaleIntensityRangePercentiles(Transform):
     .. code-block:: python
         :emphasize-lines: 11, 22
 
-        image = np.array(
+        image = torch.Tensor(
```
Comment (KumoLiu) [inline monai/transforms/intensity/array.py:1135]: The type of the `image` may not be the key point, the input dtype is the reason for the error message. It's possible that torch and numpy produce different default dtype.

Comment (matt3o) [reply monai/transforms/intensity/array.py:1135]: Hi @KumoLiu! Yes definitely, if I use `np.array` with `dtype=np.float32`, it works. Numpy choses np.int64 by default for this list. Nevertheless the return type is now a metatensor, so the prints are incorrect anyways.

Comment (KumoLiu) [reply monai/transforms/intensity/array.py:1135]: Yes, the update should be fine and just wanted to bring that up.

Comment (matt3o) [reply monai/transforms/intensity/array.py:1135]: But you are correct that this is not a bug then, it's just the wrong input dtype :) Just checked, typing is fine too, not sure what I saw there..

**Review by wyli — APPROVED** [review]

**Pull request merged**